airflow/config_templates/airflow_local_settings.py

```diff
@@ -36,6 +36,8 @@
 
 LOG_FORMAT = conf.get('core', 'LOG_FORMAT')
 
+COLORED_LOG = conf.get('core', 'COLORED_CONSOLE_LOG')
```

Comment:
There's a `getboolean` function to automatically return a boolean, useful for avoiding string checking later on.